src/routing.js

```diff
@@ -101,6 +107,10 @@ export function getAppPath({
   params,
   search = '',
 } = {}) {
+  if (!fullDao) {
```

Comment:
Can this ever happen?

Reply:
Yes, this happens in onboarding, there is no dao